lms/urls.py

```diff
@@ -426,6 +426,14 @@
         name='student_progress',
     ),
 
+    url(
+        r'^programs/{}/detail$'.format(
```

Comment:
Sorry for the back and forth on this, but let's switch this back to `r'^programs/{}/about$'.format(`.